configuration/rules-blockly.md

```diff
@@ -0,0 +1,157 @@
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+{: #blockly-introduction}
+## Introduction
+
+
+The basic idea behind the visual paradigm and representation within openHAB is based on the [Google Blockly Support](https://developers.google.com/blockly) which has been integrated and which provides the basic blocks for programming like the ones on the left and the right side of the below images
+
+{: #blockly-toolbox}
+*Blockly toolbox*
+
+![blockly-toolbox-1](images/blockly/blockly-toolbox-1.png)![blockly-toolbox-2](images/blockly/blockly-toolbox-2.png)![blockly-toolbox-3](images/blockly/blockly-toolbox-3.png)
+
+
+
+All of these provide general functionality that is not specific to openHAB itself. If you want to learn more about how to use them, search for the many blockly tutorials that are available.
+However, to leverage the full capabilities more than *50 specific blocks* have been provided that are tailored for easy access of openHAB's capabilities.
+
+This section provides a detailed description of the specific blocks and provides examples on how to use them. Note that some of the blocks (like voice, streaming or notifications) need some special setup within openHAB  - in these case links to the respective documentation is provided. It should also be mentioned that each of the blocks do have a context sensitive menu which appears upon a right click on the block itself.
+
+*Block context menu*
+
+![blockly-context.png](images/blockly/blockly-contextmenu.png)
+
+**Looking for help**
+
+A special mention should go towards the **Help** entry that links to a resource that is usually very helpful to understand the context of that particular block. Please read this information first before asking questions in the forum. *In case you ask for help please always post the respective code that is being generated.*
+
+## Before using blockly
+
+Please visit [Getting started with openHAB Blocklies](rules-blockly-before-using.html) before asking questions in the forum
+
+## Blocks
+
+### Items and Things
+
+*Items* and *Things* are the [major entities of openHAB](https://www.openhab.org/docs/concepts/) to control and monitor the home.
+
+[![Items and Things](images/blockly/blockly-items-and-things-small.png "Items and Things")
+](rules-blockly-items-things.html)
+
+See [Items & Things](rules-blockly-items-things.html) section
+
+### Timers and Delays
+
+Timers and Delays are a little more complex but add important functionality to rules. Whilst the “Wait-For”-block is straightforward, the timer blocks should be well understood before being used - they may behave differently than expected. This chapter explains what these blocks do, sometimes displaying generated code to explain what is happening behind the scenes.
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.